packages/contract-wrappers/src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -1159,30 +1162,12 @@ export class ExchangeWrapper extends ContractWrapper {
             this.getZRXAssetData(),
         );
     }
-    /**
-     * Retrieves the Ethereum address of the Exchange contract deployed on the network
-     * that the user-passed web3 provider is connected to.
-     * @returns The Ethereum address of the Exchange contract being used.
-     */
-    public getContractAddress(): string {
-        const contractAddress = this._getContractAddress(artifacts.Exchange, this._contractAddressIfExists);
-        return contractAddress;
-    }
-    /**
-     * Returns the ZRX token address used by the exchange contract.
-     * @return Address of ZRX token
-     */
-    public getZRXTokenAddress(): string {
-        const contractAddress = this._getContractAddress(artifacts.ZRXToken, this._zrxContractAddressIfExists);
-        return contractAddress;
-    }
```

Comment:
Hm... any reason to remove these? We could just return it no? It's a big breaking change to remove these... I know relayers rely on that. Do we want to force them to move to using `contract-addresses` package?

Reply:
With the changes in this PR, `zrxTokenAddress` is simply a property of `ExchangeWrapper`, so you don't have to use the `contract-addresses` package if you don't want to. Removing this method simplifies our API surface and will make `contract-wrappers` easier to use/understand. Leaving it in would mean that there are now two ways to get the address and just creates a little bit of clutter.

Given that AFAIK we don't have any mechanism for marking functions/methods as deprecated, I think we should just remove outdated code like this.

If this was the only breaking change, I would leave it in. However, we already have some breaking changes to the `contract-wrappers` package in this PR and this one is definitely on the easy side in terms of updating.